### PR TITLE
Week 12 TechniFlix W/ Nadia Lefebvre

### PR DESCRIPTION
**Pull request by Dopest0727**
Link to my deployed project: https://techniflix.netlify.app

My reflections on how this week's project turned out: Good

Things I'd like to have clarified or explained in more detail: Nothing.

### CODE REVIEW

File: code/src/components/MovieDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      width: auto;
+    }
+  }
+  @media (min-width: 1024px) {
+    flex-direction: row;
+    img{
+      height: 50vh;
+      width: auto;
+    }
+  }
```
Comment (jessand77) [inline code/src/components/MovieDetails.jsx:69]: It's not necessary to have the media query for min 1024px as they are the same as for min 767px, but it is easier to read and see differences and similarities between different sizes maybe?

File: code/src/components/MovieDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    height: auto;
+    border-radius: 10px;
+  }
+  @media (min-width: 767px) {
+    flex-direction: row;
+    img{
+      height: 50vh;
+      width: auto;
+    }
+  }
```
Comment (jessand77) [inline code/src/components/MovieDetails.jsx:62]: I had an extra look at how you managed to position this div and the elements inside. Good job with the responsiveness - it seems to work well on different devices.

File: code/src/components/MovieList.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from 'styled-components'
```
Comment (jessand77) [inline code/src/components/MovieList.jsx:3]: I like that you used styled components. It makes the code in the components much easier to follow, I think.

**Review by jessand77 — COMMENTED** [review]
Your movie app looks very good and I think you have done a great job this week! Well structured code that is easy to follow.

File: code/src/components/MovieDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  min-height: 100vh;
+  background: center center;
+  background-size: cover;
+  position: relative;
+  display: flex;
+  align-items: flex-end;
+  padding: 1rem;
+`;
+
+const Button = styled.button`
```
Comment (Laura-Sjolander) [inline code/src/components/MovieDetails.jsx:17]: Nice with the styled components, as well.

File: code/src/components/MovieDetails.jsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    left: 0%;
+    transform: translate(-50%, 25%);
+  }
+  @media (min-width: 767px) {
+    font-size: 18px;
+  }
+  @media (min-width: 1024px) {
+    font-size: 20px;
+  }
+`;
```
Comment (Laura-Sjolander) [inline code/src/components/MovieDetails.jsx:44]: Responsive page that works great!

File: code/src/components/NotFound.jsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+                <span>Back to Movies</span>
+            </Button>
+        </Link>
+        <NotFoundMessage>
+            Ooops, nothing to see here 
+            <FaSadCry />
+        </NotFoundMessage>
+    </NotFoundSection>
+  )
+}
```
[the remark this answers is not in the document]
Comment (Laura-Sjolander) [reply code/src/components/NotFound.jsx:64]: Yes, very good to have a "Not found" page

File: code/src/index.css [hunk 7]
```diff
@@ -1,13 +1,110 @@
...
+  overflow: hidden;
+  width: 1em;
+  height: 1em;
+  border-radius: 50%;
+  margin: 72px auto;
+  position: relative;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation: load6 1.7s infinite ease, round 1.7s infinite ease;
```
Comment (Laura-Sjolander) [inline code/src/index.css:38]: I wonder if the page loads to fast on Netlify, I never get to see the animation (loading spinner) thats in the Loading component ;) 
.

**Review by Laura-Sjolander — COMMENTED** [review]
Nice work guys. Well structured and easy to read. Specially liked that you had the NotFound page and the styled components. Liked the design on the "Back to movies" button, since it´s easy to see on any background whether the background is dark or light.

**Pull request closed without merging**